pkg/controller/pravegacluster/pravegacluster_controller.go

```diff
@@ -249,9 +253,13 @@ func (r *ReconcilePravegaCluster) deploySegmentStore(p *pravegav1alpha1.PravegaC
 }
 
 func (r *ReconcilePravegaCluster) syncClusterSize(p *pravegav1alpha1.PravegaCluster) (err error) {
-	err = r.syncSegmentStoreSize(p)
-	if err != nil {
-		return err
+	/*this condition is to stop syncSegmentstore version from running when we are updapting the ss from version below 07
```

Comment:
Could we just say in the comment " We skip calling syncSegmentStoreSize() during upgrade/rollback from version 07"

Reply:
done